README.md

```diff
@@ -1,70 +1,71 @@
-[![Coverage Status](https://img.shields.io/coveralls/bitcoincashjs/bitcoincashjs.svg?style=flat-square)](https://coveralls.io/r/bitcoincashjs/bitcoincashjs)
+[![NPM Package](https://img.shields.io/npm/v/bitcoincashflow.svg?style=flat-square)](https://www.npmjs.org/package/bitcoincashflow)
+[![Build Status](https://img.shields.io/travis/BitcoinDB/BitcoinCashFlow.svg?branch=master&style=flat-square)](https://travis-ci.org/BitcoinDB/BitcoinCashFlow)
+[![Coverage Status](https://img.shields.io/coveralls/BitcoinDB/BitcoinCashFlow.svg?style=flat-square)](https://coveralls.io/r/BitcoinDB/BitcoinCashFlow)
 
-[![NPM Stats](https://nodei.co/npm/bitcoincashjs.png?downloads=true)](https://nodei.co/npm/bitcoincashjs/)
+[![NPM Stats](https://nodei.co/npm/BitcoinDB/BitcoinCashFlow.png?downloads=true)](https://nodei.co/npm/BitcoinCashFlow/)
 
 ## About Bitcoin Cash
 
 Bitcoin Cash is peer-to-peer electronic cash for the Internet. It is fully decentralized, has no central bank, and requires no trusted third-parties to operate. Bitcoin Cash is the continuation of the Bitcoin project, upgraded with consensus rules that allow it to grow and scale.
 
-## About BitcoinCash.js
+## About BitcoinCashFlow
 
-BitcoinCash.js is the first JavaScript library specifically made for Bitcoin Cash. It supports all major Bitcoin Cash uses cases right out of the box, keeping up-to-date with the latest network upgrades. This library can be used - and is thoroughly tested - both in the back-end (Node.js) and the front-end (web browsers).
+BitcoinCashFlow is a JavaScript library for Bitcoin Cash. Our goal is to support all major Bitcoin Cash use cases, keep up-to-date with the latest network upgrades, follow modern JavaScript coding standards and best practices and maintain the highest levels of quality and reliability. This library can be used both in the back-end (Node.js) and the front-end (web browsers).
 
-BitcoinCash.js is a fork from [bitcore-lib](https://github.com/bitpay/bitcore-lib/), which is an extremely easy-to-use and well-tested JavaScript library for Bitcoin developed by Bitpay, Inc. However, as consensus rules between BTC and BCH become more and more incompatible, BitcoinCash.js will not add support for functionality specific to BTC, such as SegWit or the bech32 address format, and will continue to support all Bitcoin Cash uses cases as first-class citizens.
+BitcoinCashFlow is a fork of [bitcore-lib](https://github.com/bitpay/bitcore-lib/), which is an extremely easy-to-use and well-tested JavaScript library for Bitcoin developed by Bitpay, Inc. However, as consensus rules between BTC and BCH become more and more incompatible, BitcoinCashFlow will not add support for functionality specific to BTC, such as SegWit or the bech32 address format, and will continue to support all Bitcoin Cash uses cases as first-class citizens.
```

Comment:
I am not sure if we need this paragraph. Since this is a BCH library it should be obvious that other chains are not supported?

Reply:
I agree. We should either adapt or remove the paragraph completely